vendor/composer/autoload_commands_classmap.php

```diff
@@ -0,0 +1,102 @@
+<?php
```

Comment:
@kidunot89 can you run: 

```
rm -rf vendor
composer install --no-dev
```

Then push again. We don't want to version the dev dependencies that are used for Unit tests, etc. 

Thanks!

Reply:
I'll do it asap